releasenotes/notes/tag-enrichment-processor-e10bb3718600db9b.yaml

```diff
@@ -0,0 +1,13 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+features:
+  - |
+    
+    Adds a new OpenTelemetry processor component for use by the OpenTelemetry Collector (otelcol) that will enrich the tags associated with metrics, trace spans, and logs. This initial processor is a no-op processor and support for the different signals will be added incrementally.
```

Comment:
Is there a point on adding this note if the processor is noop?

Reply:
While I don't expect it to cause any problems, the noop processor is instantiated within the collector so I figured it was worth noting. If there's a better way to note this, let me know.

Reply:
Removing.